app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -98,6 +100,8 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        window.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
```

Comment:
Where are you getting this from? Does this affect every single jetpack compose app that doesn't have this hack?

Reply:
As far as I could tell this is the only workaround. Transitions in jetpack compose are still considered experimental.

Reply:
Okay, thanks.